documentation/staging/content/quickstart/create-auxiliary-image.md

```diff
@@ -0,0 +1,164 @@
+
+The quick start guide also allows you to use a ready-made, off-the-shelf auxiliary image to get the domain up and running quickly. In that case, you can skip this section and revisit it at a later time to learn the auxiliary image creation process.
+
+
+#### Prerequisites
+1. The JAVA_HOME environment variable must be set and must reference a valid JDK 8 or 11 installation.
+
+1. Download the latest [WebLogic Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling/releases) (WDT) and [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool/releases) (WIT) installer ZIP files to a new directory; for example, use directory `/tmp/quickstart/tools`. Both WDT and WIT are required to create your Model in Image auxiliary images.
+
+   For example:
+   ```
+   $ rm -rf /tmp/quickstart/tools
+   $ mkdir -p /tmp/quickstart/tools
+   ```
+   The `rm -rf` command is included in case there's an
+   old version of the tool from a
+   previous run of this quickstart tutorial.
+
+   ```shell
+   $ cd /tmp/quickstart/tools
+   ```
+   ```shell
+   $ curl -m 120 -fL https://github.com/oracle/weblogic-deploy-tooling/releases/latest/download/weblogic-deploy.zip \
+     -o /tmp/quickstart/tools/weblogic-deploy.zip
+   ```
+   ```shell
+   $ curl -m 120 -fL https://github.com/oracle/weblogic-image-tool/releases/latest/download/imagetool.zip \
+     -o /tmp/quickstart/tools/imagetool.zip
+   ```
+
+1. To set up the WebLogic Image Tool, run the following commands:
+
+   ```shell
+   $ unzip imagetool.zip
+   ```
+   ```shell
+   $ ./imagetool/bin/imagetool.sh cache deleteEntry --key wdt_latest
+   ```
+   ```shell
+   $ ./imagetool/bin/imagetool.sh cache addInstaller \
+     --type wdt \
+     --version latest \
+     --path /tmp/quickstart/tools/weblogic-deploy.zip
+   ```
+
+   Note that the WebLogic Image Tool `cache deleteEntry` command does nothing
+   if the `wdt_latest` key doesn't have a corresponding cache entry. It is included
+   because the WIT cache lookup information is stored in the `$HOME/cache/.metadata`
+   file by default, and if the cache already
+   has a version of WDT in its `--type wdt --version latest` location, then the
+   `cache addInstaller` command would fail.
+   For more information about the WIT cache, see the
+   [WIT Cache documentation](https://oracle.github.io/weblogic-image-tool/userguide/tools/cache/).
+
+   These steps will install WIT to the `/tmp/quickstart/tools/imagetool` directory,
+   plus put a `wdt_latest` entry in the tool's cache which points to the WDT ZIP file installer.
+   You will use WIT and its cached reference to the WDT installer later in the sample for creating model images.
+
+1. Download the sample WDT model, archives and properties files to be included in the auxiliary image and put them in your `/tmp/quickstart/models` directory.
```

Comment:
Download the sample WDT model, WDT properties, and WebLogic application files to be...

...and then use the `jar` command to create a WDT archive file.  For more information, see the MII UG.

Reply:
Got it. Done.